cpp/ql/src/semmle/code/cpp/ir/dataflow/DefaultTaintTracking.qll

```diff
@@ -215,19 +215,62 @@ private predicate nodeIsBarrierIn(DataFlow::Node node) {
 }
 
 cached
-private predicate instructionTaintStep(Instruction i1, Instruction i2) {
+private predicate commonTaintStep(DataFlow::Node fromNode, DataFlow::Node toNode) {
+  instructionToInstructionTaintStep(fromNode.asInstruction(), toNode.asInstruction())
```

Comment:
To be consistent with how we interleave `instruction -> operand` steps in dataflow we should probably convert this to pure `instruction -> operand` and `operand -> instruction` flow at some point once this PR has been merged. But I'm fine with these changes as they are now!

Reply:
Yeah, I just didn't want to change more than I already had to.